backend/tests/src/tests/specs/auth/sign-in/sign-in.spec.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  ExceptionMessage,
+  HttpCode,
+  HttpErrorDto,
+  HttpStatusMessage,
+  UserSignInResponseDto,
+  UserValidationMessage,
+} from 'guruhub-shared';
+
+import { Response } from '~/lib/common/types/types';
+import { withTestData } from '~/lib/helpers/helpers';
+import {
+  apiSessionStorage,
+  authService,
+  httpService,
+} from '~/lib/services/services';
+import {
+  currentUserResponseSchema,
+  errorSchema,
+  signInResponseSchema,
+} from '~/tests/json-schemas/json-schemas';
+
+describe('Sign in tests', () => {
+  before(() => {
+    apiSessionStorage.addAndEnterSession('default');
+  });
+
+  after(() => {
+    apiSessionStorage.removeSession('default');
+  });
+
+  it('should sign in with valid data', async () => {
+    const response = (await authService.signIn(
+      testsConfig.users.student,
+    )) as Response<UserSignInResponseDto>;
+
+    response.should.have.status(200);
+    response.should.have.normalExecutionTime;
+    response.body.should.have.jsonSchema(signInResponseSchema);
+    response.body.token.should.be.equal(
+      /^[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]$/,
+    );
```

Comment:
```suggestion
    /^[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]$/.test(response.body.token).should.be.true;
```

Reply:
And extract the regex to a constant (`backend/tests/src/lib/common/constants/regex.constants.ts`)
In that file:
```typescript
const JWT_TOKEN_REGEX = /^[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]\.[a-zA-Z0-9_-+/]$/;

export { JWT_TOKEN_REGEX };
```

**And you'll need to export this constant from several files.** Take `backend/src/common/constants` folder as an example